doc/source/conf.py

```diff
@@ -394,7 +394,7 @@
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     "https://docs.python.org/": None,
-    "https://scitools.org.uk/iris/docs/latest/": None,
+    "https://scitools-iris.readthedocs.io/en/latest/": None,
```

Comment:
As @benfitzpatrick says, could you merge or rebase on master to ensure your PR only shows your changes.

Reply:
@gavinevans, do you want to say what you said to me this morning? In case it's relevant.

Reply:
Only that it isn't essential to merge or rebase now, as there's no conflicts, as the changes made in this PR to the `conf.py` file are identical to those in #1324. So the main point is that merging or rebasing would clean up this PR, a little, but it's not essential.